backend/tracim_backend/lib/core/content.py

```diff
@@ -1477,6 +1477,9 @@ def move(self,
             if new_parent and new_parent.workspace_id != item.workspace_id:
                 raise ValueError('the item should stay in the same workspace')
 
+        if not new_workspace:
+            new_workspace = new_parent.workspace
+
         if (new_workspace and new_parent) and \
```

Comment:
with the 3 lines above, you can remove the check about new_workspace here:

```
         if (new_workspace and new_parent) and \ 
             new_parent.workspace_id != new_workspace.workspace_id: 
```

Can be written:

```
         if new_parent and new_parent.workspace_id != new_workspace.workspace_id: 
```

Reply:
done